source/declarations.tex

```diff
@@ -5319,8 +5319,8 @@
 For this direct-initialization, user-defined conversions are not considered.
 \item
 Otherwise,
-the initializer expression is implicitly converted to a prvalue
-of type ``\cvqual{cv1} \tcode{T1}''.
+the initializer expression is implicitly converted to a prvalue of type \tcode{T1},
+then the type of the resulting prvalue is adjusted to ``\cvqual{cv1} \tcode{T1}''.
```

Comment:
Seems clunky to convert to one type then adjust to another. There's also a risk that cv-qualifiers in the destination type of the implicit conversion could matter in some contexts. How about:

> Otherwise, the initializer expression is implicitly converted to a prvalue of type ''*cv1* `T`'' and the temporary materialization conversion is immediately applied; the reference is bound to the result of this conversion.

or similar?

Reply:
Do you think that it is clear that "immediately" means [expr.type]/2 does not apply?